app/controllers/home_controller.rb

```diff
@@ -0,0 +1,14 @@
+class HomeController < ApplicationController
+
+
+  def index
+    @spotlight = Work.spotlight
+    # if top != nil
+    #   top = @spotlight
+    # end
+    work = Work.all
+    @albums = work.sorted_list("album")
```

Comment:
It might actually read better as just `Work.all.sorted_list("album")`, without assigning `work = Work.all`